README.md

```diff
@@ -1,73 +1,130 @@
-  - 'simplcov' 
-    - [Simplecov details](https://github.com/simplecov-ruby/simplecov)
-  - `shoulda-matchers`
-    - [Shoulda-Matchers details](https://github.com/thoughtbot/shoulda-matchers) 
-  - `orderly` 
-    - [Orderly details](https://github.com/jmondo/orderly)
-    
-  - `bootstrap` 
-    - [Bootstrap details](https://github.com/twbs/bootstrap-rubygem)
-  - `rubocop`
-    -[Rubocop details](https://github.com/rubocop/rubocop)
-  
+<div id="header" align="center">
+
+<img width="762" alt="Roe'd To Aid Logo" src="https://images.prismic.io/carbmanager/68bd7d46-5f50-413c-97fe-e37b1c816159_Header-LowFODMAP.jpg?auto=compress%2Cformat&w=0.5&auto=format&fit=max">
+</div>
+
+<h1 align="center"> Welcome to KitchenSync+</h1>
+<h2 align="center">Helping you synchronize meal planning </h2>
+
+
+
+## Table of Contents
+
+- [Project Overview](#project-overview)
+- [Learning Goals](#learning-goals)
+- [Planning Documents](#planning-documents)
+- [Illustrations](#illustrations)
+- [Technologies and Tools](#technologies-and-tools)
+- [Roadmap](#roadmap)
+- [Setup](#setup)
+- [Contributors](#contributors)
+
+## Project Overview
+
+(Want to hire the devs that worked on this project? Click [this link](#contributors) to jump to our project contributors!)
+
+This repo is the Front End part of a project built by students at [Turing School of Software and Design](https://turing.edu/). You can find the Back End repo [HERE](https://github.com/KitchenSyncPlus/KSP_be) and see set-up instructions below. The project itself is an original web app designed by combined Front End and Back End teams. Its purpose is to help individuals search for recipes and create a shopping list of ingredients that can be divvied up among members of an event or group. 
+
+Visit our Front End Site!
+- [KitchenSync+]()
+
+- [KitchenSynch+ API]()
 
-```
-2.7.4
-```
 
-* Rails version
+## Learning Goals
 
-```
-5.2.8
-```
+- Continue to gain competency with current technologies and skills aquired at Turing with focus on consuming and exposing APIs.
+- Create a Service Oriented Architecture application
+- 
+- 
+- More in depth understanding and exploration of team dynamics, communication, and project processes.
 
-* Gems included
-- `pry`
-- `rspec-rails`
-- `shoulda-matchers`
-- `faraday`
+## Planning Documents
 
-* Configuration
 
-* How to run the test suite
+[Wireframes]()
 
+[Project Spec](https://backend.turing.edu/module3/projects/consultancy/)
 
-* Deployment instructions
+## Illustrations
 
+[Deployed Link]()
 
-### Credits
-  - [Mackenzie Leng](https://github.com/kenzjoy)
-  - [Michael Marchand](https://github.com/MarchandMD)
-  - [Astrid Hecht](https://github.com/Astrid-Hecht)
-  - [Sage Skaff](https://github.com/sage-skaff)
-  - [Lucas Cowell](https://github.com/lcole37)
-  - [Kevin Ta](https://github.com/KevinT001)
+[Functionality Video]()
+
+## Technologies and Tools
+
+<img width="737" alt="Screen Shot 2022-10-05 at 7 31 33 PM" src="">
+
```

Comment:
More about images: 
There's actually a more "markdown friendly" way to add images...but this is totally a legit and fine way to do things. that is, if we actually want to add an image here....